MyString.java

```diff
@@ -18,13 +18,40 @@ public static void main(String[] args) {
 
     /** Returns the lowercase version of the given string. */
     public static String lowerCase(String str) {
-        // Replace the following statement with your code
-        return null;
+        String lowerCase = "";
+
+        for (int i = 0; i < str.length(); i++) {
+            char ch = str.charAt(i);
+            if (ch >= 65 && ch <= 90) {
+                ch += 32;
+                lowerCase += ch;
+            }
+            else {
+                lowerCase += ch;
+            }
```

Comment:
since you have the same statement in both `if` and `else` statement you can move it outside. 
Since you updated the same value to same variable (fit logically, as ch is the 'next char to concat' it will still work)